src/Models/T1_relaxometry/vfa_t1.m

```diff
@@ -53,7 +53,7 @@
     properties
         MRIinputs = {'VFAData','B1map','Mask'};
         xnames = {'M0','T1'};
-        voxelwise = 1;
+        voxelwise = 0; % Fits are still voxelwise even when this is zero
```

Comment:
This name is confusing, I agree. It has a dual purpose, so it needs to be conceptually refactored; one purpose is to distinguish between which models fit voxel-by-voxel (voxelwise 1) and to distinguish which models needs neighbouring needs spatial info (e.g. QSM), but also as a flag to to enable/disable the "View data fit", which can only be done for maps that don't need spatial info (You first select a voxel, then click that button to see the fit curve for it). However, for your case, you are not fitting voxel-by-voxel, but by setting it to 0, you are removing the functionality to view the curve fit of a single voxel. We need to set this back to 1 (I tested it, and the functionality reappears and works), and open an issue to resolve this confusion here.

Reply:
Resolved in c07c7f07b

Reply:
Nevermind; I hadn't tested actually fitting the whole volume beforehand when I switched it to 1. I see that now it actually does it voxelwise if I do so. So I guess I'll switch it back to 0, we'll simply have to live with the fact that we'll be losing the feature that we can check the fit curves of a single voxel/datapoint with this new implementation. Not a huge loss, but it was a nice educational/debugging feature.

Reply:
Reverted and resolved in a53d32d215